Makefile

```diff
@@ -120,3 +120,9 @@ install-katana:
 
 run-katana:
 	katana --chain-id test --validate-max-steps 6000000 --invoke-max-steps 14000000 --eth-gas-price 0 --strk-gas-price 0 --disable-fee
+
+run-anvil:
+	anvil --block-base-fee-per-gas 10
+
+run-katana-messaging:
```

Comment:
not sure if we will need two yet, but if we decide to always run katana with the messaging mode we will need to run anvil aswell

Reply:
I'd go for always running the two then, in the end it's for end to end testing, and anvil is fast